sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -75,6 +75,15 @@ def default_encoder(obj):
       "Object of type '%s' is not JSON serializable" % type(obj).__name__)
 
 
+def get_hashable_destination(destination):
```

Comment:
https://github.com/apache/beam/pull/8076 adds a similar change and has a docstring for this function, consider copying the docstring from that PR.

Reply:
Done.